doc-source/design-principles.rst

```diff
@@ -62,13 +62,13 @@ for:
 
     {
       "locker": {
-        "repo_url": "https://github.ibm.com/my-org/my-evidence",
+        "repo_url": "https://github.com/my-org/my-evidence",
```

Comment:
`https://github.com/my-org/my-repo` for consistency.

Reply:
I think https://github.com/my-org/my-evidence-repo would be clearer, since there's the config repo, too